app/assets/javascripts/student_profile/profile_details.jsx

```diff
@@ -1,11 +1,12 @@
 import _ from 'lodash';
 import moment from 'moment';
+import ProfileDetailsStyle from '../helpers/profile_details_style.jsx';
 
 (function() {
   window.shared || (window.shared = {});
   const merge = window.shared.ReactHelpers.merge;
   const QuadConverter = window.shared.QuadConverter;
-  const styles = window.shared.ProfileDetailsStyle;
+  const styles = ProfileDetailsStyle;
```

Comment:
Can we just remove this?

Reply:
Not really... the `const styles` is used elsewhere and looks like I can't assign to `styles` when I import because `import` is read-only. So unless I switch out `styles` with `ProfileDetailsStyle` then I guess we can't remove it.